src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/EntitySchemaProcessor.cs

```diff
@@ -108,9 +115,24 @@ public void VisitUpsert(CoreModel.Substate substateData, ReferencedEntity refere
         }
     }
 
+    public void VisitDelete(CoreModel.SubstateId substateId, ReferencedEntity referencedEntity, long stateVersion)
+    {
+        if (substateId.SubstateType == CoreModel.SubstateType.SchemaEntry)
+        {
+            var keyHex = ((CoreModel.MapSubstateKey)substateId.SubstateKey).KeyHex;
+            var schemaHash = ScryptoSborUtils.DataToProgrammaticScryptoSborValueBytes(keyHex.ConvertFromHex(), _networkId);
```

Comment:
Emm... I guess the key itself isn't equal to schema hash hex-encoded?

Reply:
yes, it isn't